css/style.css

```diff
@@ -0,0 +1,15 @@
+body {
+	background-color: skyblue;
+	text-align: center;
+	margin: 0 auto;
+}
+
+.hidden {
+	display: none;
+}
```

Comment:
Not a huge deal, but there's a bootstrap class for this :smile: 

http://getbootstrap.com/css/#helper-classes-show-hide